src/core/utils/helpers/general.ts

```diff
@@ -320,4 +320,8 @@ export const filterLoansSoonOverdue = (loans: LoanType[]) => {
 export const getManifestationType = (manifestation: Manifestation) =>
   manifestation?.materialTypes?.[0]?.specific;
 
+export const useScrollLock = (modalIds: string[]) => {
```

Comment:
Im sorry to be picky about this. But it is a convention to prefix custom hooks with "use*". So I think we should avoid using it for other functions. Suggestion: `getScrollClass` ...or whatever you find appropriate.